tests/Features/ViteBundlerTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Foundation\Vite as FoundationVite;
+use Illuminate\Support\Facades\App;
+use Stancl\Tenancy\Features\ViteBundler;
+use Stancl\Tenancy\Tests\Etc\Tenant;
+use Stancl\Tenancy\Vite as StanclVite;
+
+
+test('replaces the vite helper instance with custom class', function () {
+    $concreteBindings = App::getBindings()[FoundationVite::class]['concrete'];
+
+    expect($concreteBindings(App::getInstance()))
+        ->toBeInstanceOf(FoundationVite::class);
+
+    expect($concreteBindings(App::getInstance()))
+        ->not->toBeInstanceOf(StanclVite::class);
```

Comment:
We can just instantiate the class and later remove it from the container:
```suggestion
    $vite = app(Illuminate\Foundation\Vite::class);

    expect($vite)->toBeInstanceOf(FoundationVite::class);
    expect($vite)->not->toBeInstanceOf(StanclVite::class);
```